### PR TITLE
integration: remove xmlns:doc spec from the introspection XML

### PR DESCRIPTION
**Pull request by nojocodex**
As noted in FFY00/dbus-objects#20, feel free to reject this if you think it should be addressed in `busctl` or it's just not worth fixing.

### CODE REVIEW

File: dbus_objects/integration/__init__.py [hunk 1]
```diff
@@ -50,7 +50,7 @@ def __init__(
 
     @dbus_objects.object.dbus_method(return_names=('xml',))
     def introspect(self) -> str:  # noqa: C901
-        xml = ET.Element('node', {'xmlns:doc': 'http://www.freedesktop.org/dbus/1.0/doc.dtd'})
+        xml = ET.Element('node')
```
Comment (FFY00) [inline dbus_objects/integration/__init__.py:53]: Could you leave the original line above as a comment?

**Comment by nojocodex** [comment]
Oops, messed up the PR -- let me get rid of the signal stuff...

**Review by FFY00 — CHANGES_REQUESTED** [review]
Could you please remove the signal support patch from this PR? Thanks :smiley:

File: dbus_objects/integration/__init__.py [hunk 2]
```diff
@@ -48,7 +48,8 @@ def __init__(
 
     @dbus_objects.object.dbus_method(return_names=('xml',))
     def introspect(self) -> str:  # noqa: C901
-        xml = ET.Element('node', {'xmlns:doc': 'http://www.freedesktop.org/dbus/1.0/doc.dtd'})
+        #xml = ET.Element('node', {'xmlns:doc': 'http://www.freedesktop.org/dbus/1.0/doc.dtd'}) # See: FFY00/dbus-objects#20.
```
Comment (FFY00) [inline dbus_objects/integration/__init__.py:51]: Great! Now you just need to put a space here (`# xml = ...`) to make flake8 happy :sweat_smile:

**Review by FFY00 — APPROVED** [review]

File: dbus_objects/integration/__init__.py (hunk 2 above)
Comment (nojocodex) [reply dbus_objects/integration/__init__.py]: Haha, the joys of helpful tools ;-p

**Pull request merged**